.appveyor.yml

```diff
@@ -65,15 +65,14 @@ install:
   - set PATH=%CONDA_INSTALL_LOCN%;%CONDA_INSTALL_LOCN%\scripts;%PATH%;
   - set PYTHONUNBUFFERED=1
   # for obvci_appveyor_python_build_env.cmd
-  - conda update --all --yes
-  - conda install anaconda-client=1.6.3 --yes
-  - conda install -c conda-forge --yes obvious-ci
+  - conda config --set always_yes true
+  - conda update --all
+  - conda install anaconda-client=1.6.3
```

Comment:
I think this line should be removed completely because previously `anaconda-client` were upgraded to newer version by next line edited by you.

Reply:
But I think that's because other versions of anadonda-client were breaking stuff? I can try removing the line and seeing what happens.

Reply:
Only three lines are now executed under `anaconda-client=1.6.3`
```
conda config --prepend channels conda-forge
conda config --set show_channel_urls yes
conda config --set always_yes true
```

I do not think it does matter at all. So removing the line must not change anything.

Reply:
You were right. Some dependency hell is going here.